paseos/communication/find_next_window.py

```diff
@@ -0,0 +1,59 @@
+import pykep as pk
+from loguru import logger
+
+from .get_communication_window import get_communication_window
+
+
+def find_next_window(
+    local_actor,
+    link_name: str,
```

Comment:
this is called local_actor_communication_link_name in get_communication_window.py